From d86fcf284a75e086ce0b9b5a7e8d10d84550298e Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Tue, 04 Feb 2020 14:46:39 +0200
Subject: [PATCH] [SW-8229] update sec location according to specs

Spec changed bit order of security location efuse, otp,
flash. Changed possition of efuse with flash, this is the
location for keys, revoke bits and min security fw_version

Change-Id: If7418d713f80ae1bd003132adc9dfb9bb627f22e
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/include/arcboot.h b/include/arcboot.h
index 4f57cb3..3be2f0d 100644
--- a/include/arcboot.h
+++ b/include/arcboot.h
@@ -52,9 +52,9 @@
 extern struct arcboot_global_s arcboot_rom_global;
 
 enum security_location {
-	SPI_FLASH,
+	ASIC_EFUSE,
 	SPI_OTP,
-	ASIC_EFUSE
+	SPI_FLASH
 };
 
 typedef unsigned char u8;
